node/migrations/000004_aggregate.up.sql

```diff
@@ -0,0 +1,12 @@
+CREATE TABLE IF NOT EXISTS aggregators (
+    id SERIAL PRIMARY KEY,
+    name TEXT NOT NULL UNIQUE,
+    active BOOLEAN NOT NULL DEFAULT TRUE
+);
+
+CREATE TABLE IF NOT EXISTS global_aggregates (
+    name TEXT NOT NULL,
```

Comment:
@nick-bisonai Does the name represent pair name (e.g. `UNI-USDT`), or something else?

Reply:
yes it represents pair name